README.md

```diff
@@ -278,11 +279,39 @@ Bootstrap JS are loaded.
 
 > **Note**: In general, this is a bad idea.
 
+### Images
+
+Don't forget to remove images that are no longer used.  Try to keep the images
+sorted in the local `images/` directory, with names that naturally group related
+images together in alphabetical order; for instance prefer
+`settings-file-share.png` and `settings-proxies.png` to
+`file-share-settings.png` and `proxies-settings.png`.
+
+On Mac, when applicable capture windows rather than capture rectangular regions;
+this eliminates unpleasant background appearing in the documentation, and saves
+the editors the need to crop.  Capture windows without shadows.  To this end,
+once you pressed `Command-Shift-4`, press Option while clicking on the window.
+To disable shadows once for all, run
+
+```bash
+$ defaults write com.apple.screencapture disable-shadow -bool TRUE
+$ killall SystemUIServer  # restart it.
+```
+
+(you can restore shadows later with `-bool FALSE`).
```

Comment:
More feedback about styleguide. Overall I try to avoid parens:
* If a sentence is important it should not be inside a parens
* If a sentence is going on a tangent or is not that important, just scrap it

I understand why you would use parens, but really good writing is concise and to the point. That means that there's no place for going on tangents or half-important comments.

But if you reeeeeally want to you parens, never start a sentence with them, like you are doing here.

Reply:
Got it.